src/main/java/yasp/processors/warding/Wards.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package yasp.processors.warding;
+
+import java.util.ArrayDeque;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Queue;
+import java.util.Set;
+
+import skadistats.clarity.event.Event;
+import skadistats.clarity.event.EventListener;
+import skadistats.clarity.event.Initializer;
+import skadistats.clarity.event.Provides;
+import skadistats.clarity.model.Entity;
+import skadistats.clarity.model.FieldPath;
+import skadistats.clarity.model.CombatLogEntry;
+import skadistats.clarity.processor.entities.OnEntityCreated;
+import skadistats.clarity.processor.entities.OnEntityDeleted;
+import skadistats.clarity.processor.entities.OnEntityUpdated;
+import skadistats.clarity.processor.entities.UsesEntities;
+import skadistats.clarity.processor.gameevents.OnCombatLogEntry;
+import skadistats.clarity.processor.reader.OnTickEnd;
+import skadistats.clarity.processor.runner.Context;
+import skadistats.clarity.wire.common.proto.DotaUserMessages;
+
+/**
+ * @author micaelbergeron
+ */
+@UsesEntities
+@Provides({ OnWardCountered.class, OnWardExpired.class, OnWardPlaced.class })
+public class Wards {
+    
+    private static final Set<String> WARDS_DT_CLASSES = new HashSet<String>(Arrays.asList(
```

Comment:
yes, you'd keep using a hashset.  Unfortunately it doesn't look like set literals are in Java 8: http://stackoverflow.com/questions/10705705/how-do-i-use-collection-literals-in-java-7